tests/utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+
+def get_resource_path(filename):
```

Comment:
That's super neat, thanks for the pointer! I'll probably stick with this for now so I can fix this job asap